### PR TITLE
fixes for leadership calendar downloads

### PR DESCRIPTION
**Pull request by kave**
Filter id changes had render PDFReactor non-functional. The generator was refactored to account for these changes

## Changes

- refactored filter input ids and date formatter

## Review

- @kurtw 
- @sebworks

### CODE REVIEW

**Comment by kave** [comment]
:+1: Hotswapped code on flapjack to verify fix works. @sebworks @kurtw.

File: cfgov/v1/forms.py [hunk 1]
```diff
@@ -116,6 +116,14 @@ def clean(self, value):
                 pass
         return value
 
+class PDFFilterDateField(forms.DateField):
```
Comment (sebworks) [inline cfgov/v1/forms.py]: Can't we just combine these functions and pass in a param?

Comment (kave) [inline cfgov/v1/forms.py]: @sebworks didn't have time to look for it today, but could you push up a change to this branch?

**Comment by sebworks** [comment]
:+1:

**Pull request merged**